src/chromium/manifest.json

```diff
@@ -14,7 +14,8 @@
   "background": {
     "service_worker": "ext-background.js"
   },
+  "action": {},
```

Comment:
Please add a test to each `test-manifest.js` to simply verify this value. It would be helpful to comment in the test that an empty object  activates the browser action icon so is different from an undefined `action` value.